test/acceptor_SUITE.erl

```diff
@@ -832,6 +831,46 @@ ssl_error_eacces(_) ->
 			ok
 	end.
 
+ssl_unsupported_tlsv13_options(_) ->
+	{available, Versions} = lists:keyfind(available, 1, ssl:versions()),
+	case {lists:member('tlsv1.3', Versions), do_get_ssl_version() >= {10, 0, 0}} of
+		{true, true} ->
+			do_ssl_unsupported_tlsv13_options();
+		{false, _} ->
+			{skip, "No TLSv1.3 support."};
+		{_, false} ->
+			{skip, "No TLSv1.3 option dependency checking."}
+	end.
+
+do_ssl_unsupported_tlsv13_options() ->
+	doc("Ensure that a listener can be started when TLSv1.3 is "
+	    "the only protocol and unsupported options are present."),
+	CheckOpts = [
+		{beast_mitigation, one_n_minus_one},
+		{client_renegotiation, true},
+		{next_protocols_advertised, [<<"dummy">>]},
+		{padding_check, true},
+		{psk_identity, "dummy"},
+		{secure_renegotiate, true},
+		{reuse_session, fun (_, _, _, _) -> true end},
+		{reuse_sessions, true},
+		{user_lookup_fun, fun (_, _, _) -> error end}
```

Comment:
Shouldn't this be a tuple like `{LookupFun, Secret}` or something? It doesn't really matter here, though, as the presence of the option causes `ssl:listen` to crash, and in `ranch_ssl` it's filtered out.

Reply:
Oh, indeed... I set it straight in the last commit.

Btw, what do you call a tuple with two elements? 😁

Reply:
> Btw, what do you call a tuple with two elements? 😁

A twople? XD
Did you use `jokes:flatten` on this one? It's been deprecated you know ;)

Reply:
😜